exporter/metric/metric_test.go

```diff
@@ -38,6 +43,40 @@ var (
 	}
 )
 
+func TestExportCounter(t *testing.T) {
+	cloudMock := cloudmock.NewCloudMock()
+	defer cloudMock.Shutdown()
+
+	clientOpt := option.WithGRPCConn(cloudMock.ClientConn())
+
+	resOpt := push.WithResource(
+		resource.New(
+			kv.String("test_id", "abc123"),
+		),
+	)
+
+	pusher, err := InstallNewPipeline(
+		[]Option{
+			WithProjectID("PROJECT_ID_NOT_REAL"),
+			WithMonitoringClientOptions(clientOpt),
+			WithMetricDescriptorTypeFormatter(formatter),
+		},
+		resOpt,
+	)
+	assert.NoError(t, err)
+
+	defer pusher.Stop()
+
+	// Start meter
+	ctx := context.Background()
+	meter := pusher.Provider().Meter("cloudmonitoring/test")
+
+	// Register counter value
+	counter := metric.Must(meter).NewInt64Counter("counter-a")
+	clabels := []kv.KeyValue{kv.Key("key").String("value")}
+	counter.Add(ctx, 100, clabels...)
```

Comment:
Would be nice to add a function for metrics similar to [GetNumSpans()](https://github.com/googleinterns/cloud-operations-api-mock/blob/d3cb41cc27b43ba96ca5125afc13de3161783e94/cloudmock/cloud_mock.go#L89-L90) to be able to validate that the correct data appeared on the other end.

Reply:
Alternatively, you may add another test which will construct the exporter via https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/4866baabe07e06199c33b0a3ca8aca46bb9441d7/exporter/metric/cloudmonitoring.go#L49-L50 and invoke https://github.com/GoogleCloudPlatform/opentelemetry-operations-go/blob/4866baabe07e06199c33b0a3ca8aca46bb9441d7/exporter/metric/metric.go#L174 directly and assert that there is no error